README.md

```diff
@@ -729,6 +732,92 @@ The return value is an object with fields
   - `totalSupply` ... a non-negative integer (encoded in a string) that records the total
     supply of the token as computed by using `Mint` and `Burn` events.
 
+## Get metadata URL for a list of tokens
+
+The endpoint `v0/CIS2TokenMetadata/index/subindex` retrieves a list of token
+metadata URLs.
+
+The following parameters are supported and required
+- `tokenId`: a comma separated list of token IDs. Token IDs are hex encoded, in
+  the same format as that returned by `CIS2Tokens` endpoint. An empty string is interpreted
+  as a single token with an empty ID.
+
+The return value is a JSON list of objects with fields
+- `metadataURL` (required) ... a string value that contains a URL returned by
+  the contract. The client should do validation that this is a usable URL, since
+  it is purely up to the smart contract to return this value.
+- `tokenId` (required) ... the token ID as in the query
+- `metadataChecksum` (optional) ... if `null` then no checksum is included in
+  the contract. Otherwise it is a hex string that contains the SHA256 hash of the **data at the URL**.
+
+An example query is
+```
+v0/CIS2TokenMetadata/996/0?tokenId=0b5000b73a53f0916c93c68f4b9b6ba8af5a10978634ae4f2237e1f3fbe324fa,1209fe3bc3497e47376dfbd9df0600a17c63384c85f859671956d8289e5a0be8,1209fe3bc3497e47376dfbd9df0600a17c63384c85f859671956d8289e5a0be8
+```
+
+and an example response is
+```json
+[
+  {
+    "metadataChecksum": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+    "metadataURL": "https://some.example/token/0B5000B73A53F0916C93C68F4B9B6BA8AF5A10978634AE4F2237E1F3FBE324FA",
+    "tokenId": "0b5000b73a53f0916c93c68f4b9b6ba8af5a10978634ae4f2237e1f3fbe324fa"
+  },
+  {
+    "metadataChecksum": null,
+    "metadataURL": "https://some.example/token/1209FE3BC3497E47376DFBD9DF0600A17C63384C85F859671956D8289E5A0BE8",
+    "tokenId": "1209fe3bc3497e47376dfbd9df0600a17c63384c85f859671956d8289e5a0be8"
+  },
+  {
+    "metadataChecksum": null,
+    "metadataURL": "https://some.example/token/1209FE3BC3497E47376DFBD9DF0600A17C63384C85F859671956D8289E5A0BE8",
+    "tokenId": "1209fe3bc3497e47376dfbd9df0600a17c63384c85f859671956d8289e5a0be8"
+  }
+]
+```
+
+## Get token balance for an account address
+
+The endpoint `v0/CIS2TokenBalance/index/subindex/accountAddress` retrieves the
+balance of tokens for the given account address.
+
+The following parameters are supported and required
+- `tokenId`: a comma separated list of token IDs. Token IDs are hex encoded, in
+  the same format as that returned by `CIS2Tokens` endpoint. An empty string is interpreted
+  as a single token with an empty ID.
+
+The return value is a JSON list of objects with fields
+
+- `balance` (required) ... a string that contains the balance as a decimal
+  number. The balance is always non-negative, but can be very large, and will
+  not fit into a 64-bit integer in general, so unbounded integer types should be
+  used.
```

Comment:
Maybe also mention that the metadata contains the info of how to display this number. i.e how many decimals

Reply:
Done